### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by arturobernalg**
* Simplify conditions
* Make final var

### CODE REVIEW

**Comment by garydgregory** [comment]
Please rebase on master. Recent changes should allow all builds to be green including Java 16 and 17-EA.

**Comment by arturobernalg** [comment]
> Please rebase on master. Recent changes should allow all builds to be green including Java 16 and 17-EA.

Hi @garydgregory 
done
TY

File: src/main/java/org/apache/commons/compress/archivers/zip/ZipArchiveEntry.java [hunk 1]
```diff
@@ -777,7 +777,7 @@ public String getName() {
     @Override
     public boolean isDirectory() {
         final String n = getName();
-        return n != null && n.endsWith("/");
+        return n.endsWith("/");
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/compress/archivers/zip/ZipArchiveEntry.java:780]: It does seem that `getName()` can never be null, so maybe just `return getName().endsWith("/");` ? Same for other getName() call sites?

Comment (garydgregory) [reply src/main/java/org/apache/commons/compress/archivers/zip/ZipArchiveEntry.java:780]: ping?

**Pull request merged**